src/Numerics.Tests/LinearAlgebraTests/Complex/VectorTests.cs

```diff
@@ -63,7 +63,7 @@ public void CanCloneVector()
             Assert.AreEqual(vector.Count, clone.Count);
             CollectionAssert.AreEqual(vector, clone);
         }
-        
+
```

Comment:
I have a VS extension removing the trailing whitespaces. Let me know if you want me to revert these changes.

Reply:
No, that's fine - I have similar extension installed (but apparently not touched that file since)